src/ocm/components/clusterWizard/HostDiscovery.tsx

```diff
@@ -62,12 +65,19 @@ const HostDiscovery = ({ cluster }: { cluster: Cluster }) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
     [], // just once, Formik does not reinitialize
   );
+  const { supportedPlatformIntegration } = useClusterSupportedPlatforms(cluster.id);
 
   const onSubmit: FormikConfig<HostDiscoveryValues>['onSubmit'] = async (values) => {
     clearAlerts();
 
+    const platformToIntegrate = values.usePlatformIntegration
```

Comment:
In theory if `values.usePlatformIntegration` is true, the platformIntegrion must be one of the valid values (`vsphere or nutanix`), so there is no need to double check it.